app/controllers/grade_entry_forms_controller.rb

```diff
@@ -282,9 +282,61 @@ def update_grade_entry_students
   # Download the grades for this grade entry form as a CSV file
   def csv_download
     grade_entry_form = GradeEntryForm.find(params[:id])
-    send_data grade_entry_form.get_csv_grades_report,
+    students = Student.where(hidden: false).order(:user_name)
+    grade_entry_items = grade_entry_form.grade_entry_items
+    csv_rows = []
+    # prepare first two csv rows
+    # The first row in the CSV file will contain the question names
+    row = ['']
+    grade_entry_items.each do |grade_entry_item|
+      row.push(grade_entry_item.name)
+    end
+    csv_rows.push(row)
+    # The second row in the CSV file will contain the question totals
```

Comment:
If the total is to be included, we should at a column header for it, with the "out of" in the second row.
(But as below, I only want to include this if the "show total" property is selected in the spreadsheet.)